risc0/zkvm/sdk/rust/src/lib.rs

```diff
@@ -47,8 +47,17 @@ pub mod prove;
 #[cfg(feature = "host")]
 pub mod receipt;
 pub mod serde;
-#[cfg(feature = "verify")]
-pub mod verify;
 
-// TODO: get this from the cirgen circuit
-const CODE_SIZE: usize = 16;
+#[cfg(feature = "circuit")]
+lazy_static::lazy_static! {
+    pub static ref CIRCUIT: risc0_zkvm_circuit::CircuitImpl =
```

Comment:
Would it work to have CircuitImpl have a constant constructor so we don't have to use lazy_static?  Or maybe just "CIRCUIT: risc0_zkvm_circuit::CircuitImpl = CircuitImpl;" so we don't have to modify cirgen?

Reply:
I don't know if a constant constructor will ensure that only a single copy is resident in memory. That's the intent with the `lazy_static`